src/video/VDP.hh

```diff
@@ -1217,6 +1222,10 @@ private:
 	  */
 	byte dataLatch;
 
+	/** Direction of VRAM access for reading or writing
+	 */
+	bool writeAccess;
+
```

Comment:
Please add in this comment something like:
```
Note: this variable is _only_ used for the 'VRAM access status' debuggable.
The real VDP allows to setup a read/write VRAM address and then do the opposite out/in operation.
See the variables 'cpuVramData' and 'cpuVramReqIsRead' for more details.
```
Otherwise in a few year we may wonder why we're having both `writeAccess` and `cpuVramReqIsRead`.